traffic_ops/traffic_ops_golang/deliveryservice/deliveryservices.go

```diff
@@ -97,7 +97,7 @@ func (ds *TODeliveryService) GetType() string {
 
 // IsTenantAuthorized checks that the user is authorized for both the delivery service's existing tenant, and the new tenant they're changing it to (if different).
 func (ds *TODeliveryService) IsTenantAuthorized(user *auth.CurrentUser) (bool, error) {
-	return isTenantAuthorized(ds.ReqInfo, &ds.DeliveryServiceNullableV30)
+	return isTenantAuthorizedV30(ds.ReqInfo, &ds.DeliveryServiceNullableV30)
```

Comment:
Version-specific functions like this shouldn't be necessary (see prior comment).